caravel/assets/javascripts/explorev2/components/ChartContainer.jsx

```diff
@@ -14,6 +15,13 @@ const propTypes = {
 };
 
 class ChartContainer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      selector: `#${this.props.sliceContainerId}`,
```

Comment:
looks like this is the same change as in  https://github.com/airbnb/caravel/pull/1443/files